docs/chapters/developer_guide.tex

```diff
@@ -0,0 +1,176 @@
+	iniconfig (2.0.0)             &                 \\ \hline
+	munch (4.0.0)                 &                 \\ \hline
+	numpy (1.26.3)                &                 \\ \hline
+	packaging (23.2)              &                 \\ \hline
+	pluggy (1.3.0)                &                 \\ \hline
+	python-dateutil (2.8.2)       &                 \\ \hline
+	pytz (2023.3.post1)           &                 \\ \hline
+	tomli (2.0.1)                 &                 \\ \hline
+	zipp (3.17.0)                 &                 \\ \hline
+	cftime (1.3.0)                &                 \\ \hline
+	contourpy (1.2.0)             &                 \\ \hline
+	coverage (7.4.0)              &                 \\ \hline
+	fiona (1.9.0)                 &                 \\ \hline
+	fonttools (4.47.2)            &                 \\ \hline
+	importlib-resources (6.1.1)   &                 \\ \hline
+	cycler (0.12.1)               &                 \\ \hline
+	intel-openmp (2021.4.0)       &                 \\ \hline
+	kiwisolver (1.4.5)            &                 \\ \hline
+	pandas (1.5.3)                &                 \\ \hline
+	ordered-set (4.1.0)           &                 \\ \hline
+	mypy-extensions (1.0.0)       &                 \\ \hline
+	pathspec (0.12.1)             &                 \\ \hline
+	pillow (10.2.0)               &                 \\ \hline
+	platformdirs (4.1.0)          &                 \\ \hline
+	pyparsing (3.1.1)             &                 \\ \hline
+	pyproj (3.6.1)                &                 \\ \hline
+	pyqt5-qt5 (5.15.2)            &                 \\ \hline
+	pyqt5-sip (12.13.0)           &                 \\ \hline
+	pytest (7.4.4)                &                 \\ \hline
+	shapely (2.0.2)               &                 \\ \hline
+	tbb (2021.7.1)                &                 \\ \hline
+	typing-extensions (4.9.0)     &                 \\ \hline
+	zstandard (0.22.0)            &                 \\ \hline
+	black (22.12.0)               &                 \\ \hline
+	geopandas (0.14.2)            &                 \\ \hline
+	matplotlib (3.8.2)            &                 \\ \hline
+	mkl (2021.4.0)                &                 \\ \hline
+	nuitka (1.9.7)                &                 \\ \hline
+	pyqt5 (5.15.10)               &                 \\ \hline
+	netcdf4 (1.6.5)               &                 \\ \hline
+	pytest-cov (4.1.0)            &                 \\ \hline
+	teamcity-messages (1.32)      &                 \\ \hline
+\end{longtable}
+
+\section{Tooling}
+The tool is using:
+\begin{enumerate}
+\item \keyw{anaconda} / \keyw{miniconda} /\keyw{conda} to setup its python environment. 
+\item \keyw{poetry} to install the python packages for the environment
+\item \keyw{VSCode} to edit python files, run the created (unit)tests, see/visualize the test coverage
+\end{enumerate}
+
+\subsection{Utility / install scripts}
+We use several other batch scripts.
```

Comment:
Might this chapter be more part of the Configuration chapter?

Reply:
for now i want to keep it here, after merge we will get probably same or similar feedback from Bert. Lets get a chapter with sections in the user manual and later decide to move / replace where needed. Good idea, but lets progress first.